docs/_source/practical_guides/fine_tune.md

```diff
@@ -117,6 +124,63 @@ Even though its generated internally, you can get the card by calling the `gener
 argilla_model_card = trainer.generate_model_card("my_model")
 ```
 
+##### Upload your models to Huggingface Hub
+
+
```

Comment:
The main difference between spaCy and the other frameworks is written at the end of the section:

---
```python
# spaCy based models:
repo_id = output_dir

# Every other framework:
repo_id = "organization/model-name"  # for example: argilla/newest-model

trainer.push_to_huggingface(repo_id, generate_card=True)
```

Due to the spaCy behavior when pushing models, the repo_id is automatically generated internally, you need to pass the path to where the model was saved (the same `output_dir` variable you may pass to the `train` method), and it will work out just the same.

---

From the user perspective the only difference is that you can't write the repo_id but point to the path where the model was saved (and that it relies in `spacy_huggingface_hub`). What should we add, maybe rewrite it to be more clear?

Reply:
I meant potentially needing to install `spacy_huggingface_hub` too.

Reply:
Sure! I'll add a note

Reply:
You can see it here: https://github.com/argilla-io/argilla/pull/3976/commits/e4a792ce5dc1fb4e339be3c70769c296ed64dd22